locust/test/test_main.py

```diff
@@ -2057,3 +2202,49 @@ def mytask(self):
                 self.assertEqual(status_code, 42)
             self.assertNotIn("Traceback", master_stderr)
             self.assertIn("failed to send heartbeat, setting state to missing", master_stderr)
+
+
+#     def test_interrupt_in_custom_message_handler(self):
```

Comment:
A test that doesn't work with the new stuff? I'm guessing needs to be removed or rewritten?

Reply:
actually that was just accidentally commited. I'll remove it.